models.py

```diff
@@ -307,9 +307,12 @@ def get_task_state(self):
 
     def get_default_impact_title(self):
         layer_name = '%s on %s' % (
-            self.hazard_layer.name,
-            self.exposure_layer.name
+            self.hazard_layer.title,
+            self.exposure_layer.title
         )
+        if self.aggregation_layer:
+            additional_name = 'around %s' % self.aggregation_layer.title
```

Comment:
Same reason as above, use:

```suggestion
            additional_name = 'around {aggregation}'.format(aggregation=self.aggregation_layer.title)
```